README.md

```diff
@@ -1,75 +1,87 @@
 
-# Prerequisite for onboarding to the automated pipeline:
+## Prerequisite for onboarding to the automated pipeline:
 
-## Setup Prerequisites before load test infra setup:
 Prerequisite script creates Service Connection, App Id, Service Principal and KeyVault. KeyVault has certificate and client secret.
 Steps to execute Prerequisite script:
 
-  1.	Set working directory to Scripts folder where Prerequisite.ps1 resides.
-  2.	Run below command.
+1. Set working directory to Scripts folder where Prerequisite.ps1 resides.
+2. Run below command.
+
+   .\Prerequisite -subscriptionId {Azure Subscription Id} -resourceGroupName {Resource Group Name} -keyVaultName {KeyVault Name} -location {Location} -certName {Certificate Name} -servicePrincipalName {Service Principal Name} -tenantId {Microsoft Tenand Id} -serviceConnectionName {Service Connection Name} -organizationName 'https://microsoftit.visualstudio.com' -projectName 'OneITVSO' -ServicePrincipalSecret {Service Principal Sercret Name}
+
+3. After execution of Prerequisite script, Search Service Principal Name in Azure Active Directory and fetch App Id.
+4. Onboard App Id to resource group with Contributor role.
 
-      .\Prerequisite  -subscriptionId {Azure Subscription Id} -resourceGroupName {Resource Group Name} -keyVaultName {KeyVault Name} -location {Location}  -certName {Certificate Name}  -servicePrincipalName  {Service Principal Name} -tenantId  {Microsoft Tenand Id}  -serviceConnectionName {Service Connection Name}  -organizationName 'https://microsoftit.visualstudio.com' -projectName 'OneITVSO'  -ServicePrincipalSecret {Service Principal Sercret Name}
-        
-  3. After execution of Prerequisite script, Search Service Principal Name in Azure Active Directory and fetch App Id.
-  4. Onboard App Id to resource group with Contributor role.  
-  
 ## JMeter test scripts:
-  1.	create the test suite with the help of how to setup JMeter test plan(https://jmeter.apache.org/usermanual/build-web-test-plan.html).
-  2.	Check in the JMX file and supporting files in a repository
-## AKS setup 
-  1.	Create  AKS cluster with the help of how to create a AKS cluster(https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal)
-  2.	Provide access to a Service Principal Name which would be used to run the JMX file in the cluster.
+
+1. create the test suite with the help of how to setup JMeter test plan(https://jmeter.apache.org/usermanual/build-web-test-plan.html).
+2. Check in the JMX file and supporting files in a repository
+
+## AKS setup
+
+1. Create AKS cluster with the help of how to create a AKS cluster(https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal)
+2. Provide access to a Service Principal Name which would be used to run the JMX file in the cluster.
 
 ## Steps to onboarding for the pipeline:
```

Comment:
'Steps to onboard and execute the pipeline' instead of 'Steps to onboarding for the pipeline'

Reply:
done